WTWSMS/history/titles/d_vasconic_pagan_reformed.txt

```diff
@@ -0,0 +1,4 @@
+475.1.1= {
```

Comment:
One year before game start. That's only there to ensure the title does not break, as it was missing a History file.